docs/ibc/proposals.md

```diff
@@ -39,3 +39,48 @@ The substitute client is used as a "stand in" while the subject is on trial. It
 a substitute client *after* the subject has become frozen to avoid the substitute from also becoming frozen. 
 An active substitute client allows headers to be submitted during the voting period to prevent accidental expiry 
 once the proposal passes. 
+
+# How to revive an expired client with a governance proposal
+
+See also the relevant documentation: [ADR-026, IBC client recovery mechanisms](../architecture/adr-026-ibc-client-recovery-mechanisms.md)
+
+### Preconditions 
+- The chain is updated with ibc-go >= v1.1.0.
+- Recovery parameters are set to `true` for the Tendermint light client (this determines if a governance proposal can be used). If the recovery parameters are set to `false`, recovery will require modifying the migration code. In this case the easiest thing to do is probably just to mint back the tokens during migration in the next hub upgrade. 
```

Comment:
```suggestion
- Recovery parameters are set to `true` for the Tendermint light client (this determines if a governance proposal can be used). If the recovery parameters are set to `false`, recovery will require custom migration code. 
```
Lets handle custom migration code when the problem arises. It is difficult to know the optimal fix